src/helpers/index.tsx

```diff
@@ -123,7 +123,14 @@ export const isTestnet = (networkId: NetworkId) => {
     NetworkId.FANTOM_TESTNET,
     NetworkId.POLYGON_TESTNET,
     NetworkId.TESTNET_RINKEBY,
+    NetworkId.TESTNET_GOERLI,
   ];
 
   return testnets.includes(networkId);
 };
+
+export const isChainEthereum = (chainId: number, includeTestnets: boolean): boolean => {
```

Comment:
I'm going to make one last change to give this named params... just cause I like named params... 👍